src/server/index.js

```diff
@@ -0,0 +1,241 @@
+//Load Environment Variables
+const path = require('path');
+require('../../environments/envLoader.js');
+try {
+  require('newrelic');
+} catch (e) { }
+
+
+//Load Express
+const express = require('express');
+const server = express();
+const PORT = 3002;
+
+//Load React
+const React = require('react');
+const {renderToString} = require('react-dom/server');
+const About = require('../shared/About.jsx').default;
```

Comment:
line 17? Imports the React Component from the client. The .default is just there because the require statement is wrapping the component in an object for some reason.